test/integration/src/main/java/io/pravega/test/integration/compatibility/CompatibilityChecker.java

```diff
@@ -224,8 +224,13 @@ private void checkStreamDelete() {
     }
 
     public static void main(String[] args) throws DeleteScopeFailedException {
+        String uri = System.getProperty("controllerUri");
+        if (uri == null) {
+            log.error("Enter correct gradle with correct controller URI. i.e : ./gradlew compatibilityCheck -Du=\"tcp://localhost:9090\"");
```

Comment:
This doesn't need to be executed only from gradle. So, a message like: "Input correct controller URI (e.g., "tcp://localhost:9090")" may be enough,